ckanext/hierarchy/plugin.py

```diff
@@ -141,6 +140,17 @@ def before_dataset_search(self, search_params):
             # remove include_children from the filter-list - we have a checkbox
             g.fields_grouped.pop('include_children', None)
 
+        if 'groups' in search_params['fq']:
+            group_selected = model.Group.get(search_params['fq'].split('"')[1])
```

Comment:
Does it have any sense at all? what about `fq='tags:"keyword" groups:"123"'?

Reply:
Fixed